File: src/py-dev/LiENa/LiENaStructure/LiENaMessage/LienaChannelClosedMessage.py
from LiENaStructure.LiENaMessage.LienaMessage import LienaMessage


class LienaChannelClosedMessage(LienaMessage):
    def __init__(self, message_id, target_id, origin_id, timestamps, dlc):
        LienaMessage.__init__(self, message_id, target_id, origin_id, timestamps, dlc)
        self.count = 0

File: src/cxx-dev/LiENa/lienaStructure/lienaMessage/lienaChannelClosedMessage.cpp
#include "lienaChannelClosedMessage.h"

lienaChannelClosedMessage::lienaChannelClosedMessage(unsigned long long messageId,
                                                     unsigned int targetId,
                                                     unsigned int originId,
                                                     unsigned long long timestampes,
                                                     unsigned int DLC) : lienaMessage(messageId, targetId, originId, timestampes, DLC)
{

}
